### DISCUSSION TITLE
Model stays loaded in GPU memory indefinetly after updating docker

### DISCUSSION THREAD

**Discussion by digitalpipelines** [open-webui/open-webui#3410] [General]
I'm running the docker version of v0.3.5. I upgraded from a previous version (I'm not certain but it was only about a month old). Since upgrading I noticed the addition of the JWT Expiration setting and by default it was set at -1, since I don't want it to stay loaded I tried changing it to "30m" as given in the example but it does not release it from the GPU memory and stays loaded indefinitely even though I change the setting. I have also tried to restart ollama after making the change and that did not help.

**Comment by justinh-rahb** [comment]
JWT expiration has ***ZERO*** to do with GPU memory. Go read the docs, thoroughly, front to back.
